src/chapters/14-provider-components.md

```diff
@@ -280,11 +274,11 @@ Here, we're invoking `<Rentals::Filter>` similar to how we've invoked other comp
 
 However, the main difference here is the use of `as |results|` when we are invoking our `<Rentals::Filter>` component. Incidentally, this new syntax goes hand-in-hand with the `{{yield this.results}}` syntax we were introduced to in the component template.
 
-The `as |results|` syntax might look a little new to us, but it isn't the first time that we've seen the this feature in action. Back when we first learned about the `{{#each}}` syntax, which we use to loop over a collection, we wrote something like this: `{{#each @items as |item|}}...some content here...{{/each}}`.
+The `as |results|` syntax might look a little new to us, but it isn't the first time that we've seen this feature in action. Back when we first learned about the `{{#each}}` syntax, which we use to loop over a collection, we wrote something like this: `{{#each @items as |item|}}...some content here...{{/each}}`.
 
 When we use this syntax, we are passing a block&mdash;the `...some content here...` in our example&mdash;to `{{#each}}`. Ember will iterate through the array we provided (`@items`) and render our block _once per item_ in the array.
 
-Inside of our block, we need to be able to access the current item _somehow_. The `{{#each}}` helper provides the item to our block via the `as |item|` declaration, which creates a local variable `item`, also known as a *[block parameter][TODO: link to block parameter]*.. In other words, as we iterate through `@items`, we will have access to the current item that we're looping over through the block parameter (`item`) The block parameter is only accessible from within inside of the block. Ember will fill in the block parameter with the current item of the iteration, and it will do this each time that it calls our block.
+Inside of our block, we need to be able to access the current item _somehow_. The `{{#each}}` helper provides the item to our block via the `as |item|` declaration, which creates a local variable `item`, also known as a *[block parameter][TODO: link to block parameter]*. In other words, as we iterate through `@items`, we will have access to the current item that we're looping over through the block parameter (`item`). The block parameter is only accessible from inside the block. Ember will fill in the block parameter with the current item of the iteration, and it will do this each time that it calls our block.
```

Comment:
```suggestion
Inside of our block, we need to be able to access the current item _somehow_. The `{{#each}}` syntax provides the item to our block via the `as |item|` declaration, which creates a local variable `item`, also known as a *[block parameter][TODO: link to block parameter]*. In other words, as we iterate through `@items`, we will have access to the current item that we're looping over through the block parameter (`item`). The block parameter is only accessible from inside the block. Ember will fill in the block parameter with the current item of the iteration, and it will do this each time that it renders our block.
```